todoCLI/todoCLI.js

```diff
@@ -0,0 +1,153 @@
+const rl = require(`readline`);
+const fs = require(`fs`);
+
+const menu= `(V) View ✏︎ (n) New ✏︎ (cX) Complete  ✏︎  (dX) Delete ✏︎ (q) Quit\n`;
+
+console.log(`\n\nWelcome to Todo CLI!\n----------------------`);
+const interface = rl.createInterface ( {
+	input: process.stdin,
+	output: process.stdout
+});
+
+waitingForUser();
+
+function waitingForUser () {
+  interface.question(menu, (command) => {
+    let number = parseFloat(command.replace(/[cd]/ , ''))
+    command = command.replace(number, '')
+    switch (command) {
+      case `q`:
+        process.exit();
+        break;
+      case `V`:
+        viewToDoList();
+        break;
+      case `n`:
+        addTask();
+        break;
+      case `c`:
+        completeTask(number);
```

Comment:
This needs some validation to make sure that a number was provided (also in `d`)